src/lib.rs

```diff
@@ -28,6 +28,7 @@ pub mod transports;
 pub mod types;
 
 pub use api::{Web3Main as Web3, ErasedWeb3};
+pub use helpers::CallResult;
```

Comment:
(and `CallResult::new` everywhere isn't great either, maybe a `lazy_decode` helper function as a constructor would improve readability)